05/script.js

```diff
@@ -0,0 +1,201 @@
+{
+     // Filter arr
+     const filterRange = (arr, from, to) => {
+          return arr.filter((el) => el >= from && el <= to)
+     }
+
+     const arr = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+
+     console.log(filterRange(arr, 3, 7));
+
+}
+{
+     // Sort arr
+     const arr = [324, 32423, -3242, 544, 0, 23, -454, 22, 4];
+     const sortArr = (arr) => {
+          return arr.sort((a, b) => {
+               return a - b
+          });
+     }
+
+     // -3242, -454, 0, 4, 22, 23, 324, 544, 32423
+     console.log(sortArr(arr));
+}
+{
+     // Sort arr str
+     const arr = ["4534", "a", "bb", "sdfds", "", " ", "r4rdv-"];
+     const sortArr = (arr) => {
+          return arr.sort((a, b) => {
+               return a.length - b.length
+          });
+     }
+     // '', 'a', ' ', 'bb', '4534', 'sdfds', 'r4rdv-'
+
+     console.log(sortArr(arr));
+
+}
+{
+     // Calculate average age of users
+     // older then 17 and younger then 55
+
+     const arr = [
+          { name: "John", age: 25 },
+          { name: "Pete", age: 30 },
+          { name: "Mary", age: 29 },
+          { name: "Taras", age: 25 },
+          { name: "Kate", age: 74 },
+          { name: "Chris", age: 14 },
+          { name: "Alan", age: 5 },
+          { name: "Boris", age: 55 },
+          { name: "Elizabeth", age: 48 },
+     ];
+
+     const averageAge = (arr) => {
+          const filter = arr.filter((el) => el.age > 17 && el.age < 55)
+          const average = (filter.map((el) => el.age)
```

Comment:
you don't need map here
`const averageAge = (arr) => {
  const filteredArr = arr.filter(
    (profile) => profile.age > 17 && profile.age < 55
  );
  const sum = filteredArr.reduce((acc, profile) => acc + profile.age, 0);

  return sum / filteredArr.length;
};`